### ISSUE TITLE
iOS build fails after updating the library

### ISSUE THREAD

**Issue by anagar23** [blackuy/react-native-twilio-video-webrtc#294]
### Steps to reproduce
1. I am using this library for the past one month.
2. After updating it to the latest one, gives me this error.
3. I have added it like this in my package.json - `"react-native-twilio-video-webrtc": "git+https://github.com/blackuy/react-native-twilio-video-webrtc.git"`

### Expected behavior
Should work as earlier

### Actual behavior
Gives Compile-time error

### Environment
- **Node.js version**: v10.4.1
- **React Native version**:0.61.1
- **React Native platform + platform version**: iOS 13.0/11.4

### react-native-twilio-video-webrtc
**Version**: "master"

Below are the errors - 

* Implicit declaration of function 'kTVIDefaultAVAudioSessionConfigurationBlock' is invalid in C99
* Unknown receiver 'TwilioVideoSDK'; did you mean 'TwilioVideo'?

**Comment by deshanm** [comment]
After upgrading `pod install` shows this error

```
[!] CocoaPods could not find compatible versions for pod "TwilioVideo":
  In snapshot (Podfile.lock):
    TwilioVideo (= 2.10.2, ~> 2.10.1)

  In Podfile:
    react-native-twilio-video-webrtc (from `../node_modules/react-native-twilio-video-webrtc`) was resolved to 1.0.2-1, which depends on
      TwilioVideo (~> 3.2.3)


You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * changed the constraints of dependency `TwilioVideo` inside your development pod `react-native-twilio-video-webrtc`.
   You should run `pod update TwilioVideo` to apply changes you've made.
```

**Comment by deshanm** [comment]
> After upgrading `pod install` shows this error
> 
> ```
> [!] CocoaPods could not find compatible versions for pod "TwilioVideo":
>   In snapshot (Podfile.lock):
>     TwilioVideo (= 2.10.2, ~> 2.10.1)
> 
>   In Podfile:
>     react-native-twilio-video-webrtc (from `../node_modules/react-native-twilio-video-webrtc`) was resolved to 1.0.2-1, which depends on
>       TwilioVideo (~> 3.2.3)
> 
> 
> You have either:
>  * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
>  * changed the constraints of dependency `TwilioVideo` inside your development pod `react-native-twilio-video-webrtc`.
>    You should run `pod update TwilioVideo` to apply changes you've made.
> ```

They have upgraded the IOS platform version - 
https://github.com/blackuy/react-native-twilio-video-webrtc/blob/master/react-native-twilio-video-webrtc.podspec#L16 

I had to upgrade the Pod platform version  in `ios/Podfile`
`platform :ios, '11.0'`

**Comment by anagar23** [comment]
Thanks, it worked

**Comment by marcos-mercuri-tw** [comment]
Hi,
I have the same problem. By changing the `platform` value in the `Podfile` to `'11.0'`, Does that mean that the application will only work in devices with iOS 11 or higher?
Thanks

**Comment by majorb22** [comment]
Is there a way, to install a previous version?
 I tried
`"react-native-twilio-video-webrtc": "git+https://github.com/blackuy/react-native-twilio-video-webrtc.git#a95e57fd2ed9670333d90a1ba8d47a2518216f36",`

bit it didn't work, as I got the same error message.

**Comment by marcos-mercuri-tw** [comment]
@majorb22 I did the same thing and it worked for me. Make sure you remove the `node_modules`.

**Comment by majorb22** [comment]
Yeah, thanks a lot, I removed then re-run npm install and it worked!

**Comment by raphaelrk** [comment]
A note for others trying to install a specific version of this library using `yarn`, the steps I used:
- `yarn cache clean`
- `rm yarn.lock`
- `yarn add https://github.com/blackuy/react-native-twilio-video-webrtc#16105c0e22b6c46dafc1e11d314b8dc02e481b87` (this is the current master commit, sub in whichever commit you were last using)
- `yarn`
- `cd ios`
- `rm -rf ./Pods/`
- `rm Podfile.lock`
- `pod install`

**Comment by AndreyPatseiko** [comment]
I used this  
`yarn add https://github.com/blackuy/react-native-twilio-video-webrtc#a95e57fd2ed9670333d90a1ba8d47a2518216f36`
command for **platform :ios, '9.0'**

**Comment by TaraSinghDanu** [comment]
> Is there a way, to install a previous version?
> I tried
> `"react-native-twilio-video-webrtc": "git+https://github.com/blackuy/react-native-twilio-video-webrtc.git#a95e57fd2ed9670333d90a1ba8d47a2518216f36",`
> 
> bit it didn't work, as I got the same error message.

Did same but getting this error now:
[!] CocoaPods could not find compatible versions for pod "TwilioVideo":
  In Podfile:
    react-native-twilio-video-webrtc (from `../node_modules/react-native-twilio-video-webrtc`) was resolved to 1.0.2-1, which depends on
      TwilioVideo (~> 2.10.1)

None of your spec sources contain a spec satisfying the dependency: `TwilioVideo (~> 2.10.1)`.

**Comment by Balasnest** [comment]
@TaraSinghDanu Do `pod repo update`

**Comment by TaraSinghDanu** [comment]
> @TaraSinghDanu Do `pod repo update`
> Latest Twilio iOS SDK works on xcode 11 only. So change minimum target support in podfile `platform :ios, '9.0'` to `platform :ios, '11.0'`

Ok, Now I have installed the latest using:
npm install https://github.com/blackuy/react-native-twilio-video-webrtc --save
changed the minimum target support in podfile platform :ios, '9.0' to platform :ios, '11.0'
pod repo udate

Now I am getting this error:

[!] CocoaPods could not find compatible versions for pod "TwilioVideo":
  In Podfile:
    react-native-twilio-video-webrtc (from `../node_modules/react-native-twilio-video-webrtc`) was resolved to 1.0.2-1, which depends on
      TwilioVideo (~> 3.2.3)

None of your spec sources contain a spec satisfying the dependency: `TwilioVideo (~> 3.2.3)`.

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

**Comment by Balasnest** [comment]
> You have either:
> out-of-date source repos which you can update with pod repo update or with pod install --repo-update.
> mistyped the name or version.
> not added the source repo that hosts the Podspec to your Podfile

**Comment by TaraSinghDanu** [comment]
> > You have either:
> > out-of-date source repos which you can update with pod repo update or with pod install --repo-update.
> > mistyped the name or version.
> > not added the source repo that hosts the Podspec to your Podfile
> 
> @TaraSinghDanu ^
@Balasnest 
This is my pod file 
platform :ios, '11.0'
require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'

target 'testchat' do
  # Pods for testchat
  pod 'FBLazyVector', :path => "../node_modules/react-native/Libraries/FBLazyVector"
  pod 'FBReactNativeSpec', :path => "../node_modules/react-native/Libraries/FBReactNativeSpec"
  pod 'RCTRequired', :path => "../node_modules/react-native/Libraries/RCTRequired"
  pod 'RCTTypeSafety', :path => "../node_modules/react-native/Libraries/TypeSafety"
  pod 'React', :path => '../node_modules/react-native/'
  pod 'React-Core', :path => '../node_modules/react-native/'
  pod 'React-CoreModules', :path => '../node_modules/react-native/React/CoreModules'
  pod 'React-Core/DevSupport', :path => '../node_modules/react-native/'
  pod 'React-RCTActionSheet', :path => '../node_modules/react-native/Libraries/ActionSheetIOS'
  pod 'React-RCTAnimation', :path => '../node_modules/react-native/Libraries/NativeAnimation'
  pod 'React-RCTBlob', :path => '../node_modules/react-native/Libraries/Blob'
  pod 'React-RCTImage', :path => '../node_modules/react-native/Libraries/Image'
  pod 'React-RCTLinking', :path => '../node_modules/react-native/Libraries/LinkingIOS'
  pod 'React-RCTNetwork', :path => '../node_modules/react-native/Libraries/Network'
  pod 'React-RCTSettings', :path => '../node_modules/react-native/Libraries/Settings'
  pod 'React-RCTText', :path => '../node_modules/react-native/Libraries/Text'
  pod 'React-RCTVibration', :path => '../node_modules/react-native/Libraries/Vibration'
  pod 'React-Core/RCTWebSocket', :path => '../node_modules/react-native/'

  pod 'React-cxxreact', :path => '../node_modules/react-native/ReactCommon/cxxreact'
  pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi'
  pod 'React-jsiexecutor', :path => '../node_modules/react-native/ReactCommon/jsiexecutor'
  pod 'React-jsinspector', :path => '../node_modules/react-native/ReactCommon/jsinspector'
  pod 'ReactCommon/jscallinvoker', :path => "../node_modules/react-native/ReactCommon"
  pod 'ReactCommon/turbomodule/core', :path => "../node_modules/react-native/ReactCommon"
  pod 'Yoga', :path => '../node_modules/react-native/ReactCommon/yoga'

  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'

  pod 'react-native-splash-screen', :path => '../node_modules/react-native-splash-screen'

  pod 'ReactNativeLocalization', :path => '../node_modules/react-native-localization'

  pod 'RNVectorIcons', :path => '../node_modules/react-native-vector-icons'


  pod 'RNCAsyncStorage', :path => '../node_modules/@react-native-community/async-storage'
  pod 'react-native-camera', :path => '../node_modules/react-native-camera'

  pod 'react-native-netinfo', :path => '../node_modules/@react-native-community/netinfo'


  pod 'react-native-simple-toast', :path => '../node_modules/react-native-simple-toast'

  pod 'react-native-video', :path => '../node_modules/react-native-video'
  pod 'react-native-maps', :path => '../node_modules/react-native-maps'
  pod 'GoogleMaps'
  pod 'Google-Maps-iOS-Utils'
  # add the Firebase pod for Google Analytics
  pod 'Firebase/Analytics'
  pod 'Firebase'
  pod 'Firebase/Messaging'
  # add pods for any other desired Firebase products
  # https://firebase.google.com/docs/ios/setup#available-pods

  pod 'react-native-fcm', :path => '../node_modules/react-native-fcm'


  source 'https://github.com/twilio/cocoapod-specs'
  pod 'TwilioChatClient', '~> 0.17.1'
  pod 'TwilioAccessManager', '~> 0.1.3'

  pod 'react-native-image-picker', :path => '../node_modules/react-native-image-picker'
  pod 'react-native-twilio-video-webrtc', path: '../node_modules/react-native-twilio-video-webrtc'








  pod 'RNAudio', :path => '../node_modules/react-native-audio'

  target 'ftestchatTests' do
    inherit! :search_paths
    # Pods for testing
  end
  use_native_modules!
end

target 'testchat-tvOS' do
  # Pods for testchat-tvOS

  target 'testchat-tvOSTests' do
    inherit! :search_paths
    # Pods for testing
  end

end

**Comment by TaraSinghDanu** [comment]
> > You have either:
> > out-of-date source repos which you can update with pod repo update or with pod install --repo-update.
> > mistyped the name or version.
> > not added the source repo that hosts the Podspec to your Podfile
> 
> @TaraSinghDanu ^

@Balasnest 
Replacing with pod 'react-native-twilio-video-webrtc', path: '../node_modules/react-native-twilio-video-webrtc'
with pod 'TwilioVideo'
in pod fie I am getting this error:
CocoaPods could not find compatible versions for pod "TwilioVideo":
  In Podfile:
    TwilioVideo

    react-native-twilio-video-webrtc (from `../node_modules/react-native-twilio-video-webrtc`) was resolved to 1.0.2-1, which depends on
      TwilioVideo (~> 3.2.3)

Specs satisfying the `TwilioVideo, TwilioVideo (~> 3.2.3)` dependency were found, but they required a higher minimum deployment target.
